keyboards/alps64/keymaps/numist/keymap.c

```diff
@@ -0,0 +1,91 @@
+#include QMK_KEYBOARD_H
+
+#define DYNAMIC_MACRO_SIZE 64
+#define DYNAMIC_MACRO_RANGE SAFE_RANGE
+#include "dynamic_macro.h"
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    return process_record_dynamic_macro(keycode, record);
+}
+
+enum macro_keycodes {
+    KC_M_FIND,
+    KC_M_AGAIN,
+    KC_M_UNDO,
+    KC_M_CUT,
+    KC_M_COPY,
+    KC_M_PASTE,
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
```

Comment:
This method of defining macros has been deprecated in favour of `process_record_user()` and `SEND_STRING()` - see [this page](https://docs.qmk.fm/#/feature_macros?id=the-new-way-send_string-amp-process_record_user).

Reply:
Updated everything but half of them are drycoded since the boards are at work.